stix_shifter_modules/error_test/README.md

```diff
@@ -0,0 +1,70 @@
+# Error Test
+
+### How to use
+
+To create an error test connector it just requires the same values as you would a stix_bundle connector, with a couple small changes. For example if you wanted to create a stix_bundle connector, the following connections and configurations would be valid:
+
+Connection:
+```
+{
+    "description": "teststix bundle",
+    "url": "https://raw.github.com/IBM/stix-shifter/master/data/cybox/1.json",
+    "name": "test stix bundle",
+    "type": "stix_bundle",
+    "class": "udi"
+}
+```
+
+Configuration:
+```
+{
+    "connectionId": "<connection-id>",
+    "name": "test name",
+    "description": "test description",
+    "type": "stix_bundle",
+    "class": "udi",
+    "acl": {
+        "users": [
+            {"id": "test.user", "role": "owner"}
+        ]
+    }
+}
+```
+
+If you wanted to create an error_test connector you could instead use the following Connection/Configurations:
+Connection:
+```
+{
+    "description": "test connector",
+    "url": "https://raw.github.com/IBM/stix-shifter/master/data/cybox/1.json",
+    "name": "test error connector",
+    "type": "error_test",
+    "class": "udi",
+    "options": {
+        "error_type": "transform_exception"
+    }
+}
+```
+
+Configuration:
+```
+{
+    "connectionId": "<connection-id>",
+    "name": "test name",
+    "description": "test description",
+    "type": "error_test",
+    "class": "udi",
+    "acl": {
+        "users": [
+            {"id": "test.user", "role": "owner"}
+        ]
+    }
+}
+```
+
```

Comment:
Can someone test this connector from CLI? If yes, is there any CLI commands that you can specify as examples? Just like we have in other connector readme https://github.com/opencybersecurityalliance/stix-shifter/tree/develop/stix_shifter_modules/qradar

Reply:
Ah I never tried. It throws exceptions all the way out though I think so it may not work. Will try and test it

Reply:
Nothing major. its just if someone in open source community wants to try it out. we should show some examples. But its not a mandatory requirements.